File: backend/jobs/src/main/kotlin/com/linkedout/jobs/model/Job.kt
package com.linkedout.jobs.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table
import java.util.UUID

@Table(name = "job")
data class Job(
        @Id
        val id: UUID,
        val title: String,
        val category: UUID
)

File: front/src/models/entities/job.ts
import { JobCategory } from '@/models/entities/jobCategory';

/**
 * A job.
 */
export type Job = {
  /**
   * The id of the job.
   */
  id: number;

  /**
   * The title of the job.
   */
  title: string;

  /**
   * The category of the job.
   */
  category: JobCategory;
};
